lib/code_corps/model/task_list.ex

```diff
@@ -30,10 +32,12 @@ defmodule CodeCorps.TaskList do
         position: 2
       }, %{
         inbox: false,
+        pull_requests: true,
         name: "In Progress",
         position: 3
       }, %{
         inbox: false,
+        done: true,
         name: "Done",
         position: 4
       }
```

Comment:
I would probably remove `inbox: false` from the others here for consistency.

Reply:
@JoshSmith remove all `inbox: false`?